plugins/nv-filters/nvidia-videofx-filter.c

```diff
@@ -79,9 +87,23 @@ struct nv_greenscreen_data {
 	/* Every nth frame is processed through the FX (where n =
 	 * processing_interval) so that the same mask is used for n consecutive
 	 * frames. This is to alleviate the GPU load. Default: 1 (process every frame).
+	 * Only used for AIGS.
 	 */
 	int processing_interval;
 	int processing_counter;
+
+	/* blur specific */
+	enum nvvfx_filter_id filter_id;
+	NvVFX_Handle handle_blur;
+	CUstream stream_blur;        // cuda stream
+	float strength;              // from 0 to 1, default = 0.5
+	NvCVImage *blur_BGR_dst_img; // dst img of blur fx (BGR, chunky, u8)
+	NvCVImage *RGBA_dst_img;     // tmp img used to transfer to texture
+	/* clang-format off */
+	NvCVImage *blur_dst_img; // dst img initialized from d3d11 texture (RGBA, chunky, u8)
+	/* clang-format on */
```

Comment:
Same here. You can turn clang-format off for the whole struct, but align the comments manually.

Reply:
fixed